src/views/home/Home.module.css

```diff
@@ -0,0 +1,23 @@
+.explanation {
+    color: var(--colors-grey800);
+    line-height: 20px;
+    font-size: 14px;
+    margin-bottom: var(--spacers-dp24);
+}
+
+.grid {
+    display: flex;
+    flex-wrap: wrap;
+    margin: calc(-1 * var(--spacers-dp8));
```

Comment:
I think adding `max-width: 800px` makes sense here, the 'card' content becomes difficult to scan/read when stretched across a widescreen monitor.

Reply:
Oh yeah, forgot about our container max-width, thanks!

Reply:
Fixed in 89087aa